_episodes/04-redirection.md

```diff
@@ -414,6 +414,13 @@ do anything all that impressive on their own, but when you start chaining
 them together, you can do some really powerful things very
 efficiently. 
 
+> # File manipulation and more practices with pipes
```

Comment:
One comment - to add another `#` to the title (level 2 header) to conform with lesson [formatting guidelines](https://carpentries.github.io/lesson-example/04-formatting/index.html).